app/src/main/java/Fragments/ProfileFragment.java

```diff
@@ -110,43 +131,70 @@ public void onStart() {
         super.onStart();
 
         // TODO: Set listener for add_photo ImageButton to call addPhoto()
-        addPhoto_ib.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                editPhoto();
-            }
-        });
+//        editPhoto_ib.setOnClickListener(new View.OnClickListener() {
+//            @Override
+//            public void onClick(View view) {
+//                editPhoto();
+//            }
+//        });
 
         // TODO: Set listener for edit_email, edit_phone_number
-    }
+        if (user != null) {
+            DatabaseReference myRootRef = FirebaseDatabase.getInstance().getReference();
```

Comment:
You should probably make the database references instance variables, and then modify them here instead of creating them here.